README.md

```diff
@@ -1,15 +1,14 @@
 # cumulus-message-adapter-python
 
-[![CircleCI](https://circleci.com/gh/nasa/cumulus-message-adapter-python.svg?style=svg)](https://circleci.com/gh/nasa/cumulus-message-adapter-python)
-[![PyPI version](https://badge.fury.io/py/cumulus-message-adapter-python.svg)](https://badge.fury.io/py/cumulus-message-adapter-python)
-
+[![CircleCI]](https://circleci.com/gh/nasa/cumulus-message-adapter-python)
+[![PyPI version]](https://badge.fury.io/py/cumulus-message-adapter-python)
 
 ## What is Cumulus?
 
 Cumulus is a cloud-based data ingest, archive, distribution and management
 prototype for NASA's future Earth science data streams.
 
-Read the [Cumulus Documentation](https://cumulus-nasa.github.io/)
+Read the [Cumulus Documentation]
```

Comment:
Seems like you unintentionally removed the link here

Reply:
I moved all links to the bottom of the md file.